.github/workflows/build.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     container: stabbles/sirius-cpu
     env:
-      SPEC: sirius@develop %gcc +tests +apps +vdwxc +fortran build_type=RelWithDebInfo ^openblas ^mpich
+      SPEC: "sirius@develop %gcc +tests +apps +scalapack +fortran build_type=RelWithDebInfo ^intel-mkl@2017: threads=openmp ^mpich"
```

Comment:
I think `threads=openmp` is not doing anything

Reply:
I think it was borrowed from some example in the past. Let's try without threads=openmp

Reply:
works

Reply:
FWIW, rocm should also be back

Reply:
oh it's already merged :laughing: